chicago/__init__.py

```diff
@@ -7,8 +7,8 @@
 
 class Chicago(Jurisdiction):
     division_id = 'ocd-division/country:us/state:il/place:chicago'
-    classification='council'
-    name = 'Chicago City Council'
+    classification='legislature'
+    name = 'Chicago'
```

Comment:
I'm generally a little confused about jurisdiction vs organization naming. I just ran this by @paultag and he said the jurisdiction should be "chicago city government" and that the classification should be government. I'm going to make that fix and set the whole thing up to run locally before I head out tonight so I can be sure that doesn't screw anything up. Sound OK?